lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
-In doing so, observability pipelines ensure that critically valuable observability data isn't locked up inside a single vendor solution provide granular controls for when, where, and how observability data can be processed.
+## How Does Sensu Work?
+Within Sensu, everything starts with the _agent_. 
+An agent is a [sidecar process][sidecar_process] deployed alongside your code, which observes your running systems.
+It publishes _events_ which pass through a customized pipeline that filters, modifies, and makes decisions about how to handle the events. 
+Most importantly it uses the same kinds of conditional logic that you do.
 
-While simple in concept, observabilty pipelines can be quite complex to build in practice.
+- "Does this event mean the whole system is down? If so, send an alert to PagerDuty, and to Slack."
+- "This event shows that the system is working again. Let's resolve that previous incident, and send an all-clear message to Slack. Also, stop storing metrics until another incident occurs." 
+- "Hmm.. these events seem related to our current marketing campaign. I bet they would be interesting to run some stats reports on. Let's filter them out and store in a time-series database for later analysis."
 
-![](img/observability_pipeline.png)
+### Sensu Data Model
 
-_FIGURE 1: a generic/conceptual observability pipeline ([source](https://bravenewgeek.com/the-observability-pipeline/))._
+Sensu provides an _integrated_ pipeline with a _consistent_ data model. 
+Operating like a control-plane, Sensu gives you a flexible means by which to direct multiple streams of observations and metrics to whatever destination you need them to go, and transforms them into whatever shape they need to be along the way.
 
-If the pseudo architecture diagram (or "marketecture diagram") looks a little bit like a [Rube Goldberg machine](https://en.wikipedia.org/wiki/Rube_Goldberg_machine), that's effectively what all DIY observabilty pipelines become, sooner or later.
-Thankfully, you don't need to cobble together collection of services to adopt an observability pipeline – Sensu Go is a turn-key observabilty pipeline that enables monitoring as code on any cloud.
+It can do this because every observation in Sensu is contained in a single flexible data structure; the _event_.
 
-![](img/sensu-observability-pipeline.png)
+**Example: A Sensu Event Data Structure**
+```json
+{
+  "metadata": {},
+  "entity": {},
+  "check": {},
+  "metrics": {},
+  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+  "timestamp": 1234567890
+}
+```
 
-_FIGURE 2: Sensu's turn-key solution enables organizations to adopt observability pipelines with ease._
+Events can contain numeric metrics, dictionaries of metadata, labels, and annotations, as well as any other payload of information that is meaningful to your system.
+Each component in the pipeline expects a stream of events as input and outputs a stream to the next component.
```

Comment:
This entire sentence feels out of place in the context of "data model".

Reply:
I see what you mean, but the purpose here is to show that the event data structure enables the pipeline architecture.